Formula/lib/libvirt-python.rb

```diff
@@ -0,0 +1,33 @@
+class LibvirtPython < Formula
+  desc "Libvirt virtualization API python binding"
+  homepage "https://www.libvirt.org/"
+  url "https://download.libvirt.org/python/libvirt-python-10.3.0.tar.gz"
+  sha256 "0333781ffef915d984f36a9b475ae8df6d01763883eefbd138d14c7591f51f2f"
+  license "LGPL-2.1-or-later"
+
+  livecheck do
+    url "https://download.libvirt.org/python/"
+    regex(/href=.*?libvirt-python[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.12" => [:build, :test]
+  depends_on "libvirt"
+
+  def pythons
+    deps.map(&:to_formula)
+        .select { |f| f.name.match?(/^python@\d\.\d+$/) }
+        .map { |f| f.opt_libexec/"bin/python" }
+  end
+
+  def install
+    pythons.each do |python|
+      system python, "-m", "pip", "install", *std_pip_args, "."
+    end
+  end
+
+  test do
+    system "python3", "-c", "import libvirt"
```

Comment:
Can we test a bit more than just importing it?

Reply:
Well, import does quite a lot since it loads extensions and dependent dynamic libraries.
On the other hand, almost anything meaningful would require communication with a daemon which may fail to start in CI for absolutely normal reasons.
Fortunately, there is a test driver which is just a dummy really.
Would such a test work for you?
https://github.com/unshare/homebrew-core/blob/f9fe00ab69469b38513953cbba1f721a9d7f7e4a/Formula/lib/libvirt-python.rb#L31